test/serverTest.js

```diff
@@ -66,12 +66,14 @@ async function door(locationid, signal){
 }
 
 describe('ODetect server', () => {
+    before(() => {
+        redis.connect()
```

Comment:
@mariocimet 
What do you think of this solution to allow us to disconnect from redis in the tests when they are done. Does it seem too clunky?

I also changed the `redis.quit()` call to `redis.disconnect()` because it seemed to like that better. I didn't play around with it too much though, so maybe `quit()` would have worked too.

Reply:
Doesn't seem very clunky at all. What do you mean to say when it seemed to like it better? Were you getting errors?

Reply:
Yes, I would get a failing test if I used `quit()`:
```
  1 failing

  1) BraveAlerterConfigurator
       "after all" hook:
     Error: Connection is closed.
      at Redis.sendCommand (node_modules/ioredis/built/redis/index.js:578:24)
      at Redis.quit (node_modules/ioredis/built/commander.js:125:21)
      at Object.quit (db/redis.js:55:18)
      at Context.<anonymous> (test/testBraveAlerterConfigurator.js:23:21)
      at processImmediate (internal/timers.js:456:21)
```
I'm not sure why, and it's not super clear to me from the docs (https://github.com/luin/ioredis/issues/73) what the difference is.